lib/nth_prime_cases.rb

```diff
@@ -0,0 +1,19 @@
+class NthPrimeCase < OpenStruct
+  def name
+    'test_%s' % description.downcase.gsub(/[ -]/, '_')
+  end
+
+  def do
```

Comment:
This is true, it mirrors a keyword in Ruby, so is a good feeling.

Let's avoid it if we can.